void tomo_db0::IoLoop::run_loop<true, true, false, false, false>()
--- PRE

+++ POST

@@ -633,7 +633,7 @@

 0bad [ 3] nopl (%rax)
 0bb0 [ 5] mov $0x1,%esi
 0bb5 [ 3] mov %rbx,%rdi
-0bb8 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0bb8 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0bbd [ 5] jmp <+0x31b>
 0bc2 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0bcd [ 3] nopl (%rax)
